vsphere/internal/vmworkflow/virtual_machine_clone_subresource.go

```diff
@@ -34,6 +36,11 @@ func VirtualMachineCloneSchema() map[string]*schema.Schema {
 			Optional:    true,
 			Description: "Whether or not to create a linked clone when cloning. When this option is used, the source VM must have a single snapshot associated with it.",
 		},
+		"instant_clone": {
```

Comment:
Should there be a:

```go
ConflictsWith: []string{"linked_clone"},
```

and visa versa?

Reply:
What should i do with this?

Reply:
I'm confident that `instant_clone` and `linked_clone` should not be able to be used at the same time for a resource; thus, use a `ConflictsWith` so they can not be used at the same time.

Ryan

Reply:
Can you check now if I have written it correctly?